WWDCFamily/Source/Map/Controllers/MapViewController.swift

```diff
@@ -1,11 +1,25 @@
 import UIKit
+import FirebaseAuthUI
 
 class MapViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        view.backgroundColor = .red
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Sign out", style: .done, target: self, action: #selector(signOut))
+        view.backgroundColor = .white
+    }
+
+    func signOut() {
+        do {
+            let authUI = FUIAuth.defaultAuthUI()
+            try authUI?.signOut()
+            NotificationCenter.default.post(name: Session.unauthorizedNotificationName, object: nil)
```

Comment:
I don't think this is a valid use case for a Notification. As I mentioned above we should have an abstraction around user authentication